### PR TITLE
dev-util/dart-sass-bin: new package, add 1.76.0

### PR DESCRIPTION
**Pull request by tastytea**
the reference implementation of Sass.

- needed for www-apps/hugo, see linked bug
- with inspiration from <https://github.com/BlueManCZ/edgets/tree/master/dev-util/dart-sass>

this is a binary package that bundles dart. i don't see a better alternative right now since dart isn't available in ::gentoo

### CODE REVIEW

**Comment by eli-schwartz** [comment]
> this is a binary package that bundles dart. i don't see a better alternative right now since dart isn't available in ::gentoo

It would be interesting to package dart (even as a bin) and then build dart-sass against it. :)

File: dev-util/dart-sass-bin/dart-sass-bin-1.76.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="${PN/-bin/}"
```
Comment (rahilarious) [inline dev-util/dart-sass-bin/dart-sass-bin-1.76.0.ebuild:6]:
```suggestion
MY_PN="${PN%-bin}"
```

**Comment by tastytea** [comment]
> It would be interesting to package dart (even as a bin) and then build dart-sass against it. :)

i was afraid someone would say this 😄

**Comment by tastytea** [comment]
> > this is a binary package that bundles dart. i don't see a better alternative right now since dart isn't available in ::gentoo
> 
> It would be interesting to package dart (even as a bin) and then build dart-sass against it. :)

the consensus here and in #gentoo-proxy-maint seems to be that having dart-bin packaged is preferable, so i'm gonna do that and close this PR in favor of a future source package.

**Pull request closed without merging**